scripts/apps/search/services/SearchService.js

```diff
@@ -524,6 +523,25 @@ export function SearchService($location, gettext, config, session) {
     };
 
     /**
+     * @ngdoc method
+     * @name search#mergeHighlightFields
+     * @public
+     * @description Merges the highlighted fields to the item
+     * @param {Object} item
+     * @return {Object}
```

Comment:
I think `@returns` is the correct form. It might be the reason it's not working? Not sure...